libs/tup-components/src/tickets/TicketDetailModal/TicketReplyForm.tsx

```diff
@@ -55,11 +72,27 @@ export const TicketReplyForm: React.FC<{ ticketId: string }> = ({
   return (
     <Formik
       enableReinitialize
+      // enable this line 25 for form validation.
+      // validationSchema={formSchema}
       initialValues={defaultValues}
-      validationSchema={formSchema}
       onSubmit={onSubmit}
     >
-      {({ isSubmitting, isValid }) => {
+      {({ values }) => {
+        const isReplyEmpty = values.text.length === 0;
+        const isResolved = ticketStatus === 'resolved';
+        const isChecked = values.status;
```

Comment:
Small thing: We should use the variable name `isResolved` across this component, instead of `isChecked`, to be more clear about what selecting the checkbox actually means.

Reply:
Ahh. `isChecked` is used to signify the checkbox is checked (user has clicked the checkbox). `ticketStatus` is passed in from the TicketModal. So that comes from the api response.

Reply:
Oh I see. I think what I'm asking for in that case is for `isChecked` to be something like `isCheckedForResolve` to indicate why we care about the checkbox's state.

Reply:
Ahh understood! Thank you, I pushed up your recommendation.